example/fibrechannel/main-fc.go

```diff
@@ -0,0 +1,20 @@
+package main
```

Comment:
maybe also add a README.md in this directory to help explain how to run it, etc...

Reply:
Added a README

Reply:
lgtm